python/ray/tune/tests/test_checkpoint_manager.py

```diff
@@ -119,7 +120,6 @@ def testBestCheckpointsWithNan(self):
         checkpoints += [
             _TuneCheckpoint(_TuneCheckpoint.PERSISTENT, 3, self.mock_result(0, 3))
         ]
-        random.shuffle(checkpoints)
```

Comment:
Why are we removing this?
Should we instead have a fixed order where the non-NaN checkpoint comes somewhere in the middle?

Reply:
As above